river/View.zig

```diff
@@ -506,6 +514,13 @@ pub fn notifyAppId(self: Self) void {
     }
 }
 
+/// Called when the view requests to be focused
+pub fn requestActicate(self: *Self) void {
+    self.current.urgent = true;
+    self.output.sendUrgentViews(self.current.tags);
```

Comment:
This is not particularly useful, I think.

In a scenario where you have an urgent view on tag 1 and then an urgent view on tag 2 appears, the second urgent tags event will make the client believe tag 1 is no longer urgent, which is wrong.

Instead there should be an `urgent_tags` field in the output state.

Reply:
My first version used a `urgent_tags` in the output state, I removed it because I thought it was not worth the complication, multiple urgent views didn't really cross my mind :/

Reply:
I tried something... not sure if it works as intended tbh, I don't know how to test this, I can't make it works as I want for now,

Reply:
I mark this as resolved since I changed a lot of things and this code is no more here but the implementation of this is not done